src/components/policy/policy_regular/src/sql_pt_representation.cc

```diff
@@ -1040,6 +1051,10 @@ bool SQLPTRepresentation::SaveSpecificAppPolicy(
       ? app_query.Bind(12, *app.second.allow_unknown_rpc_passthrough)
       : app_query.Bind(12);
 
+  app.second.encryption_required.is_initialized()
```

Comment:
Changes are also required in the SQLPTRepresentation::CopyApplication

Reply:
Added in 988d4fca0c6a9825e3e2c301905c00705bbcc901